controllers/leaderboard_controller.py

```diff
@@ -19,18 +19,32 @@ def get_data(self, ordering='DESC'):
         """
         with LeaderBoardDataAccess() as data_access:
             data = data_access.get_all()
-            return sorted(data, key=lambda d: -int(d.score) if ordering == 'DESC' else int(d.score))
 
-    def sort_data_asc(self):
-        """On click handler for sorting data in ASC order
+            if col == 'score':
+                return sorted(data, key=lambda d: -int(d.score) if ordering == 'DESC' else int(d.score))
+            else:
+                return sorted(data, key=lambda d: d.nick if ordering == 'DESC' else d.nick)
```

Comment:
the same result regardless of the `ordering` value

Reply:
changed